inspire/base/searchext/mappings/jobshidden.json

```diff
@@ -0,0 +1,119 @@
+{
```

Comment:
So yeah, I dunno how this will work nowadays WRT restricted records and elasticsearch. I presume it will be done on the post-process of Invenio before displaying results, so we could then keep all jobs in the same index and have some custom check there to restrict "hidden" ones.

I cannot recall how Jobs move to Jobs hidden, but I thought it was a manual curator thing. If it is just a random script or cronjob, I guess we could re-implement that as a daily celery task to move around jobs. But we are sure as long as the deadline date is reached, we move them to hidden? I figured it's a manual notice from the submitter saying that "hey, we hired someone" or "hey, you can remove the notice now".